web/renderer/components/pageComponents/ConnectionsPage/NewConnection/StartDoltServerForm.tsx

```diff
@@ -7,17 +7,27 @@ import {
 } from "@dolthub/react-components";
 import { useConfigContext } from "./context/config";
 import css from "./index.module.css";
-import { ReactNode } from "react";
+import { ReactNode, useState } from "react";
 
 type Props = {
-  disabled: boolean;
-  message?: ReactNode;
+  disabledForConnection: boolean;
+  disabledForConnectionMessage?: ReactNode;
 };
 
-export default function StartDoltServerForm({ disabled, message }: Props) {
+export default function StartDoltServerForm({
+  disabledForConnection,
+  disabledForConnectionMessage,
+}: Props) {
   const { state, setState, error, setErr, onStartDoltServer } =
     useConfigContext();
 
+  const [databaseName, setDatabaseName] = useState("");
```

Comment:
You should include this field as a part of `state` in `useConfigContext` instead of creating a standalone useState for it here